### PR TITLE
Replaced form type names with classnames

### PR DESCRIPTION
**Pull request by deguif**
I am targetting this branch, because it's about avoiding deprecation warnings when using Sf >= 2.8, while keeping the old behavior.

## Changelog
```markdown
## Fixed
- Use class name when referencing `Form Type` to be compatible with Symfony 2.8+
```
Closes #613

## Subject

Remove deprecation warnings when using Sf >= 2.8

### CODE REVIEW

File: Guesser/FilterTypeGuesser.php [hunk 1]
```diff
@@ -77,18 +85,21 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
                 return new TypeGuess('doctrine_orm_date', $options, Guess::HIGH_CONFIDENCE);
             case 'decimal':
             case 'float':
-                $options['field_type'] = 'number';
```
Comment (OskarStark) [inline Guesser/FilterTypeGuesser.php]: can you explain why you remove it?

File: Guesser/FilterTypeGuesser.php [hunk 2]
```diff
@@ -77,18 +85,21 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
...
-
-                return new TypeGuess('doctrine_orm_number', $options, Guess::MEDIUM_CONFIDENCE);
```
Comment (OskarStark) [inline Guesser/FilterTypeGuesser.php]: and this?

File: Guesser/FilterTypeGuesser.php (hunk 1 above)
Comment (deguif) [inline Guesser/FilterTypeGuesser.php]: Because this is the same than the case just after.

File: Guesser/FilterTypeGuesser.php (hunk 2 above)
Comment (deguif) [inline Guesser/FilterTypeGuesser.php]: Same as last response ;)

Comment (greg0ire) [inline Guesser/FilterTypeGuesser.php]: There is no break statement, which means lines 92 - 96 will be executed, and they do the same. So this was unnecessarily duplicated code if I understand correctly. @deguif : next time, do that in a separate commit (you can use `git add --patch` to do that easily).

**Comment by greg0ire** [comment]
LGTM

File: Guesser/FilterTypeGuesser.php (hunk 1 above)
Comment (OskarStark) [inline Guesser/FilterTypeGuesser.php]: indeed

File: Guesser/FilterTypeGuesser.php (hunk 2 above)
Comment (OskarStark) [inline Guesser/FilterTypeGuesser.php]: you both areright, i saw number != doctrine_orm_number at the first glympse, whcih confused me a bit ;-)

Comment (deguif) [inline Guesser/FilterTypeGuesser.php]: For sure, I will do it in another PR next time.

Comment (greg0ire) [inline Guesser/FilterTypeGuesser.php]: Another PR would be maybe a bit overkill, a separate commit is enough in this case IMO.

**Comment by deguif** [comment]
I just saw thereis another PR that has started to do the same: #613

**Comment by greg0ire** [comment]
Added "Closes #613" to your PR body

**Comment by greg0ire** [comment]
@OskarStark , I'll let you merge if you're ok with this.

**Comment by OskarStark** [comment]
> @OskarStark , I'll let you merge if you're ok with this.

I am, thank you @deguif 👍

**Comment by greg0ire** [comment]
Yeah thanks for your contribution(s)!
